console/network/src/helpers/id.rs

```diff
@@ -210,3 +212,17 @@ impl<F: FieldTrait, const PREFIX: u16> Distribution<AleoID<F, PREFIX>> for Stand
         AleoID::<F, PREFIX>(Uniform::rand(rng))
     }
 }
+
+impl<F: FieldTrait, const PREFIX: u16> Ord for AleoID<F, PREFIX> {
+    #[inline]
+    fn cmp(&self, other: &Self) -> Ordering {
+        self.0.cmp(&other.0)
```

Comment:
Note: Ord wasn't implemented because it is unnatural to order `bech32m` **strings** by their `4829384field` **field** value.

Reply:
And I was concerned implementing a string comparator would be too expensive.

Reply:
I see, that makes the third time I made an unnatural Ord proposal. Closing this PR.

In the related snarkOS PR, I'm going ahead and order by the inner field anyway, just to distinguish transactions which come in at the same nanosecond. Which is also semantically weird, but it does the job.